Doc/whatsnew/3.13.rst

```diff
@@ -1003,6 +1003,12 @@ New Features
   functions on Python 3.11 and 3.12.
   (Contributed by Victor Stinner in :gh:`107073`.)
 
+* Add :c:func:`PyUnicode_EqualToUTF8` function: compare Unicode object with
+  a :c:expr:`const char*` UTF-8 encoded bytes string and return true (``1``)
```

Comment:
```suggestion
  a :c:expr:`const char*` UTF-8 encoded string and return true (``1``)
```

Reply:
Note that "UTF-8 encoded bytes string" is used at least in 17 other places in the C API documentation and 3 times outside of it. If it is incorrect, we should fix them first.

Reply:
> If it is incorrect, we should fix them first.

I don't get this rationale. If the change makes sense, it should be used for new doc.

Fixing old doc is a separated concern. I don't see how it should be a pre-requirement to write correct doc.